src/main/appOne.java

```diff
@@ -0,0 +1,11 @@
+class Test{
```

Comment:
<b>code: if(string1 == string2) { - </b>
Use .equals() method to compare strings instead of == operator.